### ISSUE TITLE
Trevorspray User Enumeration Bug

### ISSUE THREAD

**Issue by D4rthMaulCop** [blacklanternsecurity/TREVORspray#31]
There seems to be an issue with user enumeration when using Trevorspray. No options are given when attempting to enumerate users.
Output:
```bash
[06/27/23 5:53:00] root@kali:~/trevorspray# trevorspray --users users.txt --recon example.net -v
[INFO] Command: /usr/local/bin/trevorspray --users users.txt --recon example.net -v
[INFO] User enumeration enabled with --recon and --users
[INFO] Choosing user enumeration method (skip by exporting TREVOR_userenum_method=)

[USER] Which user enumeration method would you like to use? () 
```

**Comment by 4lph4b** [comment]
bump

**Comment by TheTechromancer** [comment]
Thanks for reporting. This is fixed in 0aa2893a4c721c7e610e03a99ed3ab35678c896e.
